<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>217</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>202</width>
    <height>217</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>202</width>
    <height>217</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:rgb(90,90,90);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonTo8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>()8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTo16">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>()16</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRoot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTimes">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 71, 71);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 71, 71));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFac">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSign">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(90,90,90);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(90,90,90));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTo2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>()2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEq">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 71, 71);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 71, 71));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDiv">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTo10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>()10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMinus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(23, 133, 133);
  border: 1px solid gray;
	color:white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(0,0,0), stop: 1 rgb(23, 101, 101));
	color:white;
}</string>
    </property>
    <property name="text">
     <string>x^n</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border: 1px solid gray;
	qproperty-alignment: 'AlignVCenter | AlignRight';
	background-color:rgb(90,90,90);
	color:white;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>202</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
